openstack/container/experimental/capsules/results.go

```diff
@@ -0,0 +1,77 @@
+package capsules
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// Extract is a function that accepts a result and extracts a capsule resource.
+func (r commonResult) Extract() (*Capsule, error) {
+	var s *Capsule
+	err := r.ExtractInto(&s)
+	return s, err
+}
+
+// GetResult represents the result of a get operation.
+type GetResult struct {
+	commonResult
+}
+
+// Represents a Container Orchestration Engine Bay, i.e. a cluster
+type Capsule struct {
+	// UUID for the capsule
+	UUID string `json:"uuid"`
+
+	// ID for the capsule
+	ID int `json:"id"`
+
+	// User ID for the capsule
+	UserID string `json:"user_id"`
+
+	// Project ID for the capsule
+	ProjectID string `json:"project_id"`
+
+	// cpu for the capsule
+	CPU float64 `json:"cpu"`
+
+	// Memory for the capsule
+	Memory string `json:"memory"`
+
+	// The name of the capsule
+	Name string `json:"meta_name"`
+
+	// Indicates whether capsule is currently operational. Possible values include:
+	// Running,
+	Status string `json:"status"`
+
+	// The created time of the capsule.
+	Created string `json:"created_at"`
```

Comment:
This should be:

```go
CreatedAt time.Time `json:"created_at"`
```

If there's a problem converting the timestamp, then it should be:

```go
CreatedAt time.Time `json:"-"`
```

And then implement a custom `UnmarshalJSON` method. See this [FAQ entry](https://github.com/gophercloud/gophercloud/blob/master/FAQ.md#overriding-default-unmarshaljson-method) for an example.